pyowc/data/fetchers.py

```diff
@@ -19,6 +19,46 @@
 from ..configuration import print_cfg
 
 
+@lru_cache(maxsize=50)
+def read_mat_ctd(filepath):
+    """Cache the result of reading a ctd .mat file from disk."""
+    return read_mat(filepath)
+
+
+@lru_cache(maxsize=50)
+def read_mat_bottle(filepath):
+    """Load and transform a bottle .mat file."""
+    data = read_mat(filepath)
+
+    # if data dimensions don't match, transpose to avoid indexing issues
+    if (data['lat'].size == data['pres'].size and data['lat'].shape[1] == data['pres'].shape[1]):
```

Comment:
Can these parentheses be dropped?

Reply:
See https://github.com/euroargodev/argodmqc_owc/pull/72#discussion_r744596357